src/site/content/en/angular/creating-pwa-with-angular-cli/index.md

```diff
@@ -44,6 +44,17 @@ This command will:
 * Add the [`theme-color`](/themed-omnibox) `<meta>` tag to `index.html`.
 * Create app icons in the `src/assets` directory.
 
+## Register the service worker at startup
+
+In `app.module.ts`, make sure the service worker starts immediately so it can serve pages when offline:
+
+```typescript
+ServiceWorkerModule.register('ngsw-worker.js', {
+  enabled: environment.production,
+  registrationStrategy: 'registerImmediately',
```

Comment:
I'd suggest not recommending this strategy. By default we use `registerWhenStable` with `30000ms` timeout to make sure the app is stable (i.e. no ongoing micro/macro tasks).

It might be worth rephrasing/changing the paragraph to just discuss briefly the different registration strategies from the [docs](https://angular.io/api/service-worker/SwRegistrationOptions#registrationStrategy), but I'd say the majority of folks (95%+) would want to go with the default.

Reply:
Fair enough, but I don't think my app (https://github.com/k0swe/forester) is doing anything too exotic, and yet service worker registration takes a long time and Lighthouse can't identify it as a PWA. It was incredibly difficult to find information about how to resolve that situation.

Reply:
Maybe this experience isn't common enough to warrant adding to docs here. Is there a way to see what tasks are delaying my SW registration?

Reply:
Changed, PTAL.